packages/picker/src/Picker.ts

```diff
@@ -469,27 +484,17 @@ export class PickerBase extends SizedMixin(Focusable, { noDefaultSize: true }) {
                     : html`
                           <span hidden id="applied-label">${appliedLabel}</span>
                       `}
-                ${this.invalid && !this.pending
+                ${this.invalid && !this.pendingStateController.isPending()
```

Comment:
same as above...

Reply:
The isPending function offers dynamic flexibility in determining the pending state. Directly managing a variable to store this state would reduce this flexibility. Additionally, introducing an extra state variable requires meticulous synchronization with the host’s state changes. This imposes additional and potentially redundant responsibilities on the controller. If the state variable is not accurately updated or if certain edge cases are not handled, it can lead to inconsistencies in the UI. Therefore, maintaining the isPending function ensures a more robust and flexible approach to managing the pending state.

Reply:
I don't see a flexibility here only it adds an overhead to import the boolean value, can you create a property and import it here as `this.pendingStateController.isPending`, this is much more cleaner and faster

Reply:
If I do like this, this won't be faster because for everytime when host is updated  I manually have to set `this.isPending = this.host.pending;` so that this.pendingStateController.isPending is in sync with this.pending